multi_node_screen.sh

```diff
@@ -2,8 +2,9 @@
 
 # clean and compile the project
 echo "clean and compile the project"
-mix clean
-mix compile
+mix deps.get || exit 1 
```

Comment:
shouldn't we first clean and then get the deps?

Reply:
`Dependencies’ sources and build files are cleaned only if the --deps option is given.`

`mix clean` doesn't affect `mix deps.get`